npm/pkg/controlplane/translation/translatePolicy.go

```diff
@@ -239,6 +259,28 @@ func targetPodSelector(ns string, matchType policies.MatchType, selector *metav1
 	return podSelectorIPSets, podSelectorList
 }
 
+// PodSelector translates podSelector of spec field and NetworkPolicyPeer in networkpolicy object to trasnslatedIPSet and SetInfo.
+func podSelector(ns string, matchType policies.MatchType, selector *metav1.LabelSelector) ([]*ipsets.TranslatedIPSet, []policies.SetInfo) {
```

Comment:
where does this get called?

Reply:
did you mean to replace targetPodSelector()?

Reply:
Yes. I tried to scope this PR in the scope of `parseSelector.go`. 
I am planning to have another PR to clean up all unneeded code (e.g., `targetPodSelector` and its helper functions and namespaceSelectors) in a next PR to avoid confusion. Sorry. I may put comments about it.
I confirmed it with UT - https://github.com/Azure/azure-container-networking/blob/2a932fc65dc63aa6574da9f2a569dfd4b2bf5929/npm/pkg/controlplane/translation/translatePolicy_test.go#L1369